### PR TITLE
chore(package): update xo to version 0.17.0

### PR DESCRIPTION
**Pull request by jfmengels**
The Greenkeeper PR #43 needs some manual fixing, so here it is.

I've disabled a rule in there, waiting for a fix from here https://github.com/avajs/eslint-plugin-ava/pull/148. You may want to wait to merge this until a patch is released though, as it will happen pretty soon I'm sure.

cc @sindresorhus

### CODE REVIEW

**Comment by sindresorhus** [comment]
Fixed in latest eslint-plugin-ava. @jfmengels Can you update and fix the merge conflict?

**Comment by jfmengels** [comment]
Done!
There wasn't a merge conflict though. The "approval box" was gray because it needs a review, not because there was a merge conflict.

File: test/test.js [hunk 1]
```diff
@@ -10,6 +10,7 @@ const fileName = `${homedir()}/.hyper.js`;
 const fileContent = 'module.exports = {plugins: [], localPlugins:[]};';
 let api = require('../api');
 
+// See https://github.com/avajs/eslint-plugin-ava/pull/148
```
Comment (matheuss) [inline test/test.js]: @jfmengels can you please remove it 😄

**Comment by matheuss** [comment]
Thank you @jfmengels @sindresorhus 😄

**Comment by jfmengels** [comment]
My pleasure :)

**Pull request merged**